test/fixtures/semantic-html5-scenarios/section-with-id-roles.adoc

```diff
@@ -0,0 +1,4 @@
+[#exciting.lead.awesome]
```

Comment:
Perhaps `#exciting.awesome.feature` instead. I don't want to overload the use of the term "lead".